conda/recipes/dask-cuda/recipe.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+schema_version: 1
+
+context:
+  version: ${{ env.get("RAPIDS_PACKAGE_VERSION") }}
+  minor_version: ${{ (version | split("."))[:2] | join(".") }}
+  date_string: '${{ env.get("RAPIDS_DATE_STRING") }}'
+  head_rev: ${{ git.head_rev(".")[:8] }}
+
+package:
+  name: dask-cuda
+  version: ${{ version }}
+
+source:
+  path: ../../..
+
+build:
+  noarch: python
+  string: ${{ date_string }}_${{ head_rev }}
+  script:
+    content: |
+      python -m pip install . -vv
+  python:
+    entry_points:
+      - dask-cuda-worker = dask_cuda.cli:worker
+      - dask-cuda-config = dask_cuda.cli:config
+
+requirements:
+  host:
+    - python
+    - pip
+    - rapids-build-backend >=0.3.0,<0.4.0dev0
+    - setuptools >=64.0.0
+  run:
+    - python
```

Comment:
I don't know the `rattler-build` recipe syntax well so can't make an exact suggestion, but if this is going to be `noarch` I think we should set a floor on `python` to the lowest version we test against, something like:

```yaml
    - python >=3.10
```

I'm not sure how you want to accomplish that in CI... maybe change the `matrix_filter` in `.github/workflows` to build on the OLDEST Python instead of the latest, then rely on environment variables in the container? Figuring out a pattern for that would be part of the work for https://github.com/rapidsai/build-planning/issues/43, I think.

Reply:
I think for now I'm comfortable setting this equal to the value of `python-requires` in the `pyproject.toml`, so at least we're self-consistent across conda and pip

Reply:
Misread the above comment -- we aren't `noarch` anymore so this doesn't apply at the moment.

Setting the lower bound equal to `python-requires` is a reasonable pattern, I think, although at the moment it doesn't work because the `rattler` jinja parser sees the `-` as a minus sign and fails out.

Reply:
https://github.com/prefix-dev/rattler-build/issues/1539

Reply:
Ah yep! Now that we're deferring `noarch` to later, nothing needs to be done for this comment. We'll get a Python 3.10 package that'll get built with a `python =3.10.*` or similar constraint, and similar for Python 3.11 and 3.12.

Thanks for opening that `rattler` issue, I subscribed there.

Reply:
Just to update on the `rattler-build` issue, it's possible to escape keys with `-` and other problem characters by using square brackets to look up fields, e.g.

```yaml
py_requires: ${{ load_from_file("pyproject.toml")["project"]["requires-python"] }}
```